packages/react/src/AnchorNav/AnchorNav.module.css

```diff
@@ -331,7 +331,8 @@
     width: 100%;
     height: 100vh; /* Keep as fallback */
     height: 100dvh;
-    background: hsla(var(--base-color-scale-black-0-hsl) / 0.5);
+    /* replace with overlay-backdrop-bgColor */
+    background: #8c959f33;
```

Comment:
This is the only one i'm seeing a regression on 👇 

![Screenshot 2023-10-30 at 14 23 59](https://github.com/primer/brand/assets/13340707/5a3dbefa-a6ce-4f90-bcb0-d0ea61fb4bcf)

Think we can reduce opacity for time being. I'll push the change...

Reply:
Yeah, I sort of on a whim aligned this closer to Primer product design tokens but we can totally revert this change

Reply:
👍 Yeah if that's okay, let's go back to old one for now to minimise visual regression as much as possible. I've reverted this to black at 50% now (still as a hex value)... We can put in a separate PR down the line to change the visuals.